### DISCUSSION TITLE
couldn't find microphone connections in jamulus

### DISCUSSION THREAD

**Discussion by Mengni-ni** [jamulussoftware/jamulus#2986] [Technical Support & Help]
I followed the guide line of installing Jamulus on my MacBook, but somehow the audio settings need a bid fix and I'm here for your help, if anyone can tell me, where am I able to find or add Jamulus to microphone allowance, cuz I couldn't even see it as an option in the privacy tab, under the microphone setting allowance.

**Comment by pljones** [comment]
Please list the steps you followed to install Jamulus, starting from indicating the page you downloaded it from.

**Reply by Mengni-ni** [reply]
Ok, so first I started opening the website jamulus.io
then I clicked "Mac" for automatic downloading,
then I drag those two iOS app to application
then I double clicked the blue one and started setting 
then I opened the red one and started setting 
I did set the language to mandarin though cuz I speak mandarin...but does that effect anything?

Thanks for your response though

**Reply by Mengni-ni** [reply]
somehow it told me to set up my audio settings, and I clicked preferences choose privacy bar and then Tabs and see microphone but just didn't see an option to add Jamulus

**Comment by ann0see** [comment]
The microphone permission request should come up the first time you connect to a server. Did you already do that?

**Reply by Mengni-ni** [reply]
I haven't even noticed that, no, I don't think I have, I think I somehow missed it, is there anyway to fix this?
Thank you so much!

**Reply by ann0see** [reply]
Yes. You can reset the permission request via terminal. Probably you can follow https://apple.stackexchange.com/questions/384317/how-do-i-reset-camera-and-microphone-permission-on-macos-mojave#384318
